src/platform_impl/linux/x11/event_processor.rs

```diff
@@ -502,21 +507,19 @@ impl EventProcessor {
             }
 
             self.dnd.source_window = Some(source_window);
-            if self.dnd.result.is_none() {
-                let time = if version >= 1 {
-                    xev.data.get_long(3) as xproto::Timestamp
-                } else {
-                    // In version 0, time isn't specified
-                    x11rb::CURRENT_TIME
-                };
+            let time = if version >= 1 {
```

Comment:
Can you rewrite it with the next in mind.

```suggestion
            let time = if version == 0 {
```

Reply:
Flipped the branches of the if-expression as requested.